docs/guides/chip_tool_guide.md

```diff
@@ -778,25 +841,74 @@ following command:
 $ ./chip-tool discover commissioners
 ```
 
+<hr>
+
 ### Pairing
 
-The `pairing` command supports different means regarding Matter device
-commissioning procedure.
+The `pairing` command supports different methods for Matter device commissioning
+procedure. The three most common methods are described in the
+[Using CHIP Tool for Matter device testing](#using-chip-tool-for-matter-device-testing)
+section and are the following:
 
-Thread and Wi-Fi commissioning use cases are described in the
-[Using the CHIP Tool for Matter device testing](#using-chip-tool-for-matter-device-testing)
-section.
+-   `ble-thread` - For Thread commissioning; described under
+    [Commissioning into a Thread network over Bluetooth LE](#commissioning-into-a-thread-network-over-bluetooth-le).
+-   `ble-wifi` - For Wi-Fi commissioning; described under
+    [Commissioning into a Wi-Fi network over Bluetooth LE](#commissioning-into-a-wi-fi-network-over-bluetooth-le)
+-   `onnetwork` - For commissioning the device when it is already present in an
+    IP network; described under
+    [Commissioning with setup PIN code](#commissioning-with-setup-pin-code)
```

Comment:
Every single one of these is deprecated and ideally should not be used.  The things to use are `code-thread`, `code-wifi`, and `code` unless you are testing some very specific commissioning edge cases.

Reply:
@bzbarsky-apple why these commands are deprecated?
I understand that `code-wifi`, `code`, `code-thread`, is more realistic case for real controller (to commission device using real controller you provide the qr_code payload, manual pairing code, nfc payload), however does it mean that changes that will be applied for `code-thread` commissioning will not be applied for `ble-thread` commissioning command?

Reply:
> why these commands are deprecated?

Because they are too easy to mess up, and people should generally not use them.  To be clear, "deprecated" means "don't steer people to using these", not "going to remove them".

> does it mean that changes that will be applied for code-thread commissioning will not be applied for ble-thread commissioning command

Generally speaking, changes will be applied to both, if it makes sense.  But in practice, I know people care about making sure `code-thread` does sane things, whereas `ble-thread` the caring is a lot less....